examples/07_catalog/discovery.ipynb

```diff
@@ -0,0 +1,253 @@
+{
```

Comment:
Just ran the notebook and it seems it's outdated (still using properties for `countries` or `datasets` or `variables`. Please try to run it locally to fix those errors.